doc/doxygen/src/getting-started.md

```diff
@@ -39,7 +39,7 @@ simplest way to compile and link an application with RIOT, is to set up a
 Makefile providing at least the following variables:
 
  * `APPLICATION`: should contain the (unique) name of your application
- * `BOARD`: specifies the platform the application should be build for by
+ * `BOARD`: specifies the platform the application should be built for by
```

Comment:
I don't think past tense should be used here (as the application isn't built yet ;-)), but maybe one of our native speakers can clarify. @danpetry, @kb2ma?

Reply:
Use another verb, let's say `do`, then `should be done` seems better than `should be do`, right ?

Reply:
Mh... You are right. :smile: